sources/academy/webscraping/scraping_basics_python/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Web scraping basics for Python devs
+description: Learn how to use Python to extract information from websites in this practical course, starting from the absolute basics.
+sidebar_position: 10
+category: web scraping & automation
+slug: /scraping-basics-python
+---
+
+import DocCardList from '@theme/DocCardList';
+
+**Learn how to use Python to extract information from websites in this practical course, starting from the absolute basics.**
+
+---
+
+:::danger Work in progress
+
+This course is incomplete. As we work on adding new lessons, we would love to hear your feedback. Comment right here under each page or [file a GitHub Issue](https://github.com/apify/apify-docs/issues) to discuss a problem.
+
+:::
+
+In this course we'll use Python to create an application for watching prices. It'll be able to scrape all product pages of an e-commerce website and record prices. Data from several runs of such program would be useful for seeing trends in price changes, detecting discounts, etc.
+
+<!--
+TODO image of warehouse with some CVS or JSON exported, similar to sources/academy/webscraping/scraping_basics_javascript/images/beginners-data-collection.png, which is for some reason the same as sources/academy/webscraping/scraping_basics_javascript/images/beginners-data-extraction.png
+-->
+
+![E-commerce listing on the left, JSON with data on the right](./images/scraping.png)
+
+## What you'll do
+
+- Inspect pages using browser DevTools
+- Download web pages using the HTTPX library
+- Extract data from web pages using the Beautiful Soup library
+- Save extracted data in various formats, e.g. CSV which MS Excel or Google Sheets can open
+- Follow links programatically (crawling)
+- Save time and effort with frameworks, such as Crawlee, and scraping platforms, such as Apify
+
+## Who this course is for
+
+Anyone with basic knowledge of developing programs in Python who wants to start with web scraping can take this course. The course does not expect you to have any prior knowledge of web technologies or scraping.
+
+## Requirements
+
+- macOS, Linux or Windows machine with a web browser and Python installed
+- Familiar with Python basics: variables, conditions, loops, functions, strings, lists, dictionaries, files, classes, exceptions
+- Comfortable importing from the Python standard library, using virtual environments, and installing dependencies with `pip`
+- Familiar with how to run commands in Terminal or Command Prompt
```

Comment:
```suggestion
## Requirements

- A macOS, Linux, or Windows machine with a web browser and Python installed
- Familiarity with Python basics: variables, conditions, loops, functions, strings, lists, dictionaries, files, classes, and exceptions
- Comfort with importing from the Python standard library, using virtual environments, and installing dependencies with `pip`
- Familiarity with running commands in Terminal (macOS/Linux) or Command Prompt (Windows)
```

Slight rewrite for parallel structure in list.

Reply:
Thanks!